keras_hub/src/models/flux/flux_model.py

```diff
@@ -0,0 +1,177 @@
+import keras
+
+from keras_hub.src.models.flux.flux_layers import DoubleStreamBlock
+from keras_hub.src.models.flux.flux_layers import EmbedND
+from keras_hub.src.models.flux.flux_layers import LastLayer
+from keras_hub.src.models.flux.flux_layers import MLPEmbedder
+from keras_hub.src.models.flux.flux_layers import SingleStreamBlock
+from keras_hub.src.models.flux.flux_maths import TimestepEmbedding
+
+
+class Flux(keras.Model):
```

Comment:
This class will need to be added as `FluxBackbone`,  The backbone needs to be a functional model. 
You will then add a new task model flux_text_to_image.py. This will be configured with a preprocessor and a backbone. you would need to add a preprocessor flow - here is an example : https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/stable_diffusion_3/stable_diffusion_3_text_to_image_preprocessor.py
and a generate function like this : https://github.com/keras-team/keras-hub/blob/master/keras_hub/src/models/stable_diffusion_3/stable_diffusion_3_text_to_image.py#L126